Resources/Prototypes/XenoArch/triggers.yml

```diff
@@ -0,0 +1,262 @@
+  components:
+  - type: XATGas
+    targetGas: NitrousOxide
+  - type: XATReactive
+    reagents:
+    - NitrousOxide
+
+- type: xenoArchTrigger
+  id: TriggerFrezon
+  tip: xenoarch-trigger-tip-frezon
+  components:
+  - type: XATGas
+    targetGas: Frezon
+  - type: XATReactive
+    reagents:
+    - Frezon
+
+- type: xenoArchTrigger
+  id: TriggerRadiation
+  tip: xenoarch-trigger-tip-radiation
+  components:
+  - type: XATDamage
+    typesNeeded:
+      Radiation: 100
+  # TODO: legendary microwave trigger
+
+- type: xenoArchTrigger
+  id: TriggerPressureHigh
+  tip: xenoarch-trigger-tip-pressure-high
+  components:
+  - type: XATPressure
+    maxPressureThreshold: 385
+
+- type: xenoArchTrigger
+  id: TriggerPressureLow
+  tip: xenoarch-trigger-tip-pressure-low
+  components:
+  - type: XATPressure
+    minPressureThreshold: 50
+
+- type: xenoArchTrigger
+  id: TriggerExamine
+  tip: xenoarch-trigger-tip-examine
+  components:
+  - type: XATExamine
+
+- type: xenoArchTrigger
+  id: TriggerBruteDamage
+  tip: xenoarch-trigger-tip-brute-damage
+  components:
+  - type: XATDamage
+    groupsNeeded:
+      Brute: 150
+
+- type: xenoArchTrigger
+  id: TriggerInteraction
+  tip: xenoarch-trigger-tip-interaction
+  components:
+  - type: XATInteraction
+
+- type: xenoArchTrigger
+  id: TriggerWrenching
+  tip: xenoarch-trigger-tip-wrenching
+  components:
+  - type: XATToolUse
+    requiredTool: Anchoring
+  - type: XATExaminableText
+    examineText: xenoarch-trigger-examine-wrenching
+
+- type: xenoArchTrigger
+  id: TriggerPrying
+  tip: xenoarch-trigger-tip-prying
+  components:
+  - type: XATToolUse
+    requiredTool: Prying
+  - type: XATExaminableText
+    examineText: xenoarch-trigger-examine-prying
+
+- type: xenoArchTrigger
+  id: TriggerScrewing
+  tip: xenoarch-trigger-tip-screwing
+  components:
+  - type: XATToolUse
+    requiredTool: Screwing
+  - type: XATExaminableText
+    examineText: xenoarch-trigger-examine-screwing
+
+- type: xenoArchTrigger
+  id: TriggerPulsing
+  tip: xenoarch-trigger-tip-pulsing
+  components:
+  - type: XATToolUse
+    requiredTool: Pulsing
+  - type: XATExaminableText
+    examineText: xenoarch-trigger-examine-pulsing
+
+- type: xenoArchTrigger
+  id: TriggerTimer
+  tip: xenoarch-trigger-tip-timer
+  components:
+  - type: XATTimer
+    delay: 120 #2 minutes
+
+- type: xenoArchTrigger
+  id: TriggerBlood
+  tip: xenoarch-trigger-tip-blood
+  components:
+  - type: XATReactive
+    reagents:
+    - Blood
+    - CopperBlood
+    - InsectBlood
+    - Slime
+    - AmmoniaBlood
+    - ZombieBlood
+
+- type: xenoArchTrigger
+  id: TriggerThrow
+  tip: xenoarch-trigger-tip-throw
+  whitelist:
+    components:
+    - Item
+  components:
+  - type: XATItemLand
+
+- type: xenoArchTrigger
+  id: TriggerDeath
+  tip: xenoarch-trigger-tip-death
+  components:
+  - type: XATDeath
+
+- type: xenoArchTrigger
+  id: TriggerMagnet
+  tip: xenoarch-trigger-tip-magnet
+  components:
+  - type: XATMagnet
```

Comment:
this is just a "if you feel like it" but splitting the magnet into 'weak' and 'strong' for magboots vs salvage magnet would actually be so kino.

Reply:
i'll add it to 'polishing' issue too, ok. I felt like there should be some game-related stuff with salv magnet, more then just attracting big piles of rocks / broken ships, but it feels so tedious to program and then balance it. :sadge: So just 2 triggers feels okay man.